_includes/xlsform.md

```diff
@@ -186,6 +186,8 @@ XLSForm has a number of data type options available for meta data collection:
 | subscriberid  | IMSI (International Mobile Subscriber Identity)     |
 | simserial    | SIM serial number.     |
 | phonenumber  | Phone number (if available).      |
+| username      | Username configured (if available).  |
+| email         | Google Account configured (if available).  |
```

Comment:
This should be just "email address", the client can choose which email address to set and how!

Reply:
Oops, @danbjoseph I see you asked that question above. 😊